app/templates/buyers/award_or_cancel_brief.html

```diff
@@ -0,0 +1,75 @@
+{% extends "_base_page.html" %}
+
+{% import "toolkit/forms/macros/forms.html" as forms %}
+
+{% block page_title %}{{ brief.title or brief.lotName }} – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".buyer_dashboard"),
+        "label": "Your account"
```

Comment:
If the compare and decide team have merged their breadcrumbs work, we should build on top of the functionality they've added as otherwise when they remove the feature flag we might be missing a breadcrumb here

Reply:
Done